src/components/message/index.tsx

```diff
@@ -0,0 +1,11 @@
+interface ErrorMessageProps {
+  message?: string;
+}
+
+const ErrorMessage = ({ message }: ErrorMessageProps) => {
```

Comment:
É interessante a criação da interface de forma separada quando vc exporta ela, ou tem muitas props para receber. Nesse caso, não está errado, mas seu código seria mais enxuto só com a tipagem do message não?

```suggestion
const ErrorMessage = (message?: string) => {
```

Reply:
Um vídeo seria melhor mesmo, anotada a dica!!
E a alteração já foi feita em relação ao código, valeu!